pypeit/core/flux_calib.py

```diff
@@ -1131,23 +1170,26 @@ def zeropoint_to_throughput(wave, zeropoint, eff_aperture):
 
 def zeropoint_qa_plot(wave, zeropoint_data, zeropoint_data_gpm, zeropoint_fit, zeropoint_fit_gpm, title='Zeropoint QA', axis=None, show=False):
     """
-    QA plot for zeropoint plotting
+    QA plot for zeropoint
 
     Parameters
     ----------
-    wave
-    zeropoint_data
-    zeropoint_data_gpm
-    zeropoint_fit
-    zeropoint_fit_gpm
-    title
-    order
-    axis
-    show
-
-    Returns
-    -------
-
+    wave: `numpy.ndarray`
```

Comment:
`numpy.ndarray`  -->  `numpy.ndarray`_  for a few of these

Reply:
fixed